ledger/participant-integration-api/src/main/scala/platform/apiserver/LedgerFeatures.scala

```diff
@@ -17,4 +18,6 @@ case class LedgerFeatures(
     contractIdFeatures: ExperimentalContractIds = ExperimentalContractIds.defaultInstance,
     committerEventLog: ExperimentalCommitterEventLog =
       ExperimentalCommitterEventLog.of(eventLogType = CENTRALIZED),
+    explicitDisclosureUnsafe: ExperimentalExplicitDisclosure =
```

Comment:
All experimental features are per definition unsafe. Since this is an internal flag, I don't see what we are gaining by painting it red at every level, but that's just my 2c

Reply:
good point. Addressed